app/src/test/java/com/example/marvelapp/CharacterListViewModelTest.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.marvelapp
+
+class CharacterListViewModelTest {
```

Comment:
the ViewModel tests should be inside a ViewModel package

Reply:
done